launch/cartographer.launch

```diff
@@ -15,15 +15,14 @@
   limitations under the License.
 -->
 <launch>
-    <arg name="robotont_ns" default="robotont"/>
-    <node name="cartographer_node" pkg="cartographer_ros"
-        type="cartographer_node" args="
-        -configuration_directory $(find robotont_demos)/config
-        -configuration_basename robotont_cartographer_localization_2d.lua"
-        output="screen">
-        <remap from="odom" to="$(arg robotont_ns)/odom"/>
-        <remap from="echoes" to="scan" />
```

Comment:
@mmaao Kas selle remap-i eemaldamine midagi katki ei tee? Ei leia esimese hooga kohta kus kartograaferile öeldakse et loe scan-i...

ok ise küsin ise vastan: lk 12 https://buildmedia.readthedocs.org/media/pdf/google-cartographer-ros/latest/google-cartographer-ros.pdf:
If you use only one sensor_msgs/LaserScan topic, it is expected to be named scan. If you have more,
they should be named scan_1, scan_2 etc. . .